packages/react-email/source/commands/dev.ts

```diff
@@ -111,7 +111,9 @@ const createAppFiles = async () => {
         : `${SRC_PATH}/pages/${page.title}`;
 
       if (page.dir) {
-        createDirectory(`${SRC_PATH}/pages/${page.dir}`);
+        return createDirectory(`${SRC_PATH}/pages/${page.dir}`).then(() =>
+          fs.promises.writeFile(location, page.content),
+        );
```

Comment:
If the map is made async:
`const pageCreation = pages.map(async (page) => {`

Then this can be simplified to:
`await createDirectory(`${SRC_PATH}/pages/${page.dir}`);`

There's no need for .then and the repeat write file line if await is used

Reply:
Yeah, for sure.